src/t2iapi/biceps/localizedtext.proto

```diff
@@ -0,0 +1,31 @@
+/*
+This Source Code Form is subject to the terms of the MIT License.
+Copyright (c) 2024 Draegerwerk AG & Co. KGaA.
+
+SPDX-License-Identifier: MIT
+*/
+
+syntax = "proto3";
+
+package t2iapi.biceps;
+
+option java_package = "com.draeger.medical.t2iapi.biceps";
+option java_outer_classname = "LocalizedTextProto";
+
+import "t2iapi/biceps/localizedtextcontent.proto";
+import "t2iapi/biceps/localizedtextref.proto";
+import "google/protobuf/wrappers.proto";
+import "t2iapi/biceps/referencedversion.proto";
+import "t2iapi/biceps/localizedtextwidth.proto";
+
+/*
+Represents pm:LocalizedText (defined in IEEE Std 11073-10207-2017).
+*/
+message LocalizedTextMsg {
+  LocalizedTextContentMsg localized_text_content = 1;
+  LocalizedTextRefMsg ref_attr = 2;
```

Comment:
no need for this type indirection, it's a StringValue

Reply:
done

Reply:
done

Reply:
Not really though, you made this into a string, not StringValue

Reply:
done